### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: src/stores/config.js [hunk 1]
```diff
@@ -12,11 +12,13 @@ exports.setUp = function (basePath, blobStore, locks) {
           if (err) {
             return callback(err)
           }
+          var result
           try {
-            callback(null, JSON.parse(config))
```
Comment (diasdavid) [inline src/stores/config.js]: Why doesn't this work?

Comment (diasdavid) [inline src/stores/config.js]: If JSON.parse throws, the error is automatically catch by the try/catch clause, here is a simplier example:

```JavaScript
try {
  checkIfRuns(JSON.parse('sdadadas:asda'))
} catch (e) {
  console.log('=> error')
}

function checkIfRuns (arg1) {
  console.log('->', arg1)
}
```

```bash
$ node try-catch.js
=> error
```

Comment (ralphtheninja) [inline src/stores/config.js]: It works but if *callback* throws that will also be catched.

Comment (diasdavid) [inline src/stores/config.js]: I see, good catch! thank you @ralphtheninja :)

Comment (ralphtheninja) [inline src/stores/config.js]: My first code contribution. Yay! :dancers:

Comment (jbenet) [inline src/stores/config.js]: :fireworks: :tada: :smile:

**Pull request merged**